pkg/util/cmdutil/diag.go

```diff
@@ -24,11 +24,29 @@ import (
 
 var snk diag.Sink
 
+var globalColorization = colors.Always
+
+// GetGlobalColorization gets the global setting for how things should be colored.
+// This is helpful for the parts of our stack that do not take a DisplayOptions struct.
+func GetGlobalColorization() colors.Colorization {
```

Comment:
it's somewhat the reverse.  the pulumi command just goes and sets this, so that it's available for anyone to go find if they need.  

That's what i meant in my comment about us not needing to really pass along any 'color' option anymore.  There is always just a well known place it can be found.